AI_interface.py

```diff
@@ -138,7 +132,7 @@ def __init__(self):
     def train_model(self, starting_train_step=0):
         os.environ["TF_FORCE_GPU_ALLOW_GROWTH"] = "true"
         gpus = tf.config.list_physical_devices('GPU')
-        ray.init(num_gpus=len(gpus), num_cpus=16)
+        ray.init(num_gpus=len(gpus), num_cpus=3)
```

Comment:
This change should not be here. If needed, num_cpus should be at least a configurable parameter